fedimint-fountain/src/lib.rs

```diff
@@ -0,0 +1,115 @@
+//! Fountain encoder/decoder for splitting and recombining byte payloads
+//!
+//! This is an inline fork of the fountain module from ur-rs,
+//! providing rateless fountain codes for efficient data transmission.
```

Comment:
What's their license? We should include it here and possibly mention the original authors